src/workerd/api/tests/js-rpc-test.js

```diff
@@ -455,6 +456,23 @@ export class MyService extends WorkerEntrypoint {
       })()
     );
   }
+
+  testImportedWaitUntil() {
+    // Initialize globalWaitUntilPromise to a promise that will be resolved in a waitUntil task
+    // later on. We'll perform a cross-context wait to verify that the waitUntil task actually
+    // completes and resolves the promise.
+    let resolve;
+    globalWaitUntilPromise = new Promise((r) => {
+      resolve = r;
+    });
+
+    waitUntil(
+      (async () => {
+        await scheduler.wait(100);
+        resolve();
+      })()
+    );
```

Comment:
nit: I believe you could simplify this down to...
```suggestion
    waitUntil(scheduler.wait(100).then(resolve));
```